README.md

```diff
@@ -106,6 +108,15 @@ checkers using `tox`:
 tox
 ```
 
+To test Elastic Inference with Accelerator, you will need an AWS account, publish your built image to ECR repository and run the following command:
```

Comment:
can you provide an example of the command needed to run the test?

Reply:
"To test against Elastic Inference, you will..."

Reply:
I'll update these as well as the above comments.